sdccc/src/test/java/com/draeger/medical/sdccc/util/MessageGeneratingUtilTest.java

```diff
@@ -176,6 +178,45 @@ public void testGetLocalizedTextPayloadTooLarge() throws Exception {
         assertFalse(observer.isInvalid());
     }
 
+    /**
+     * Verifies that 413 SOAPFault payload too large errors do not invalidate the test run.
+     *
+     * @throws Exception on any exception
+     */
+    @Test
+    public void testGetLocalizedTextPayloadTooLarge2() throws Exception {
+        final var mockHostedService = mock(HostedServiceProxy.class, Mockito.RETURNS_DEEP_STUBS);
+        final SoapMessage soapMessage = mock(SoapMessage.class, RETURNS_DEEP_STUBS);
+        final JAXBElement<?> body = mock(JAXBElement.class);
+        when(soapMessage.getOriginalEnvelope().getBody().getAny().get(0)).thenReturn(body);
+        final SoapFaultException soapFaultException =
```

Comment:
Since you only need the cause of the SoapFaultException, you could also mock it. This would eliminate some imports and simplify the test.

Reply:
Done.